Test/git-issues/git-issue-1384.dfy

```diff
@@ -0,0 +1,9 @@
+// RUN: %dafny /compile:0 /env:0 /dprint:- "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method Test()
+{
+    
+    var f : ((int,int)) -> int := ( x: (int,int)) => 1;
+
```

Comment:
For styling, use 2 spaces for indents, no space between an identifier and the following `:`.
```suggestion    
    var f: ((int,int)) -> int := (x: (int,int)) => 1;
```
(Unless the example is intentionally different from the pretty-printed output.)